src/main/java/uk/gov/hmcts/reform/hmc/service/InboundQueueServiceImpl.java

```diff
@@ -142,12 +144,27 @@ private void updateHearingAndStatus(Long hearingId, HearingResponse hearingRespo
         }
     }
 
+    @Transactional
+    private void updateHearingAndStatus(Long hearingId, SyncResponse syncResponse) {
+        Optional<HearingEntity> hearingResult = hearingRepository.findById(hearingId);
+        if (hearingResult.isPresent()) {
+            HearingEntity hearingToSave = hmiHearingResponseMapper.mapHmiSyncResponseToEntity(
+                syncResponse,
+                hearingResult.get()
+            );
+            HearingEntity hearingEntity = hearingRepository.save(hearingToSave);
+            HmcHearingResponse hmcHearingResponse = getHmcHearingResponse(hearingEntity);
+            messageSenderToTopicConfiguration.sendMessage(hmcHearingResponse.toString());
+            if (hearingEntity.getStatus().equals(HearingStatus.EXCEPTION.name())) {
+                // TODO: Raise alert for Dynatrace (approach TBC)
+            }
+        }
+    }
+
     private HmcHearingResponse getHmcHearingResponse(HearingEntity hearingEntity) {
-        Optional<HearingResponseEntity> hearingResponseEntity =
-            hearingEntity.getHearingResponses()
-                .stream().max(Comparator.comparing(HearingResponseEntity::getHearingResponseId));
-        return hmiHearingResponseMapper
-            .mapEntityToHmcModel(hearingResponseEntity
-                                     .orElseThrow(() -> new BadRequestException("bad request")), hearingEntity);
+        Optional<HearingResponseEntity> hearingResponseEntity = hearingEntity.getLatestHearingResponse();
```

Comment:
Think what was previously here was incorrect, checking by response ID instead of latest based on version/time? It also threw an exception in the scenarios where there haven't yet been any hearing responses, which will always be the case for `POST /hearing`.